### PR TITLE
Document `rlp decode` subcommand

### LINKED ISSUE #1575: Add the `decode` subcommand
Add a rlp decode command that will output the contents of the BFT encoded RLP extra data.
Support decoding IBFT legacy, IBFT2 and QBFT extra data formats.

For more info see: https://github.com/hyperledger/besu/pull/6895

### PR DESCRIPTION
**Pull request by alexandratran**
Document the `besu rlp decode` subcommand for decoding IBFT 2.0 and QBFT RLP-encoded `extraData` strings. Minor fixes to related content for consistency. Fixes #1575.

Preview: https://besu-docs-5cuvz4xt6-hyperledger.vercel.app/development/private-networks/reference/cli/subcommands#decode

### CODE REVIEW

File: docs/private-networks/reference/cli/subcommands.md [hunk 1]
```diff
@@ -65,23 +65,65 @@ The configuration file has two nested JSON nodes. The first is the `genesis` pro
...
+
+</Tabs>
+
+Decodes the RLP hexadecimal string used as `extraData` in an
+[IBFT 2.0](../../how-to/configure/consensus/ibft.md#extra-data) or
+[QBFT](../../how-to/configure/consensus/qbft.md#extra-data) genesis file into a validator list.
+
+This subcommand takes the following options:
+
+- `from` - The file containing the RLP hexadecimal string to decode.
```
Comment (macfarla) [inline docs/private-networks/reference/cli/subcommands.md:104]: default is standard input

File: docs/private-networks/reference/cli/subcommands.md [hunk 2]
```diff
@@ -65,23 +65,65 @@ The configuration file has two nested JSON nodes. The first is the `genesis` pro
...
+- `to` - The file to write the decoded validator list to.
```
Comment (macfarla) [inline docs/private-networks/reference/cli/subcommands.md:105]: default is standard output

File: docs/private-networks/reference/cli/subcommands.md [hunk 3]
```diff
@@ -95,13 +137,17 @@ cat extra_data.json | besu rlp encode > rlp.txt
...
+Encodes a validator list into an RLP hexadecimal string to use as `extraData` in an
+[IBFT 2.0](../../how-to/configure/consensus/ibft.md#extra-data) or
+[QBFT](../../how-to/configure/consensus/qbft.md#extra-data) genesis file.
 
-- `IBFT_EXTRA_DATA` - The IBFT 2.0 genesis file includes the `IBFT_EXTRA_DATA` type in the [`extraData`](../../how-to/configure/consensus/ibft.md#extra-data) property.
+This subcommand takes the following options:
 
-- `QBFT_EXTRA_DATA` - The QBFT genesis file includes the `QBFT_EXTRA_DATA` type in the [`extraData`](../../how-to/configure/consensus/qbft.md#extra-data) property.
+- `from` - The file containing the validator list to encode.
+- `to` - The file to write the RLP-encoded hexadecimal string to.
```
Comment (macfarla) [inline docs/private-networks/reference/cli/subcommands.md:147]: default is std in/ std out

**Review by macfarla — COMMENTED** [review]
to and from are also optional - think we should explicitly say default is to use standard in / standard out - as shown in the examples already

**Review by macfarla — APPROVED** [review]

**Review by joaniefromtheblock — APPROVED** [review]
LGTM

**Pull request merged**